src/libtest/stats.rs

```diff
@@ -332,6 +332,7 @@ pub fn winsorize<T: Float + FromPrimitive>(samples: &mut [T], pct: T) {
 
 /// Returns a HashMap with the number of occurrences of every element in the
 /// sequence that the iterator exposes.
+#[cfg(not(stage0))]
```

Comment:
for that bug in libtest after the `Hash` definition was changed. I've the same fix in the oibit patch, I guess we can wait for that one to land and remove this one.